src/utils/permission/index.js

```diff
@@ -113,7 +114,7 @@ export const requestExternalStoragePermission = async () => {
         ? PERMISSIONS.ANDROID.READ_EXTERNAL_STORAGE
         : PERMISSIONS.IOS.PHOTO_LIBRARY
     );
-    if (requestResult === RESULTS.GRANTED) {
+    if (requestResult === RESULTS.GRANTED || requestResult === RESULTS.LIMITED) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The message returned when the permission is `RESULTS.LIMITED` should be different from when it's `RESULTS.GRANTED`. The user should be informed that they have limited access, not full access as the current message implies.
```diff
-        message: StringConstant.cameraPermissionGranted,
+        message: StringConstant.cameraPermissionLimited,
```
Please make sure to define `StringConstant.cameraPermissionLimited` with an appropriate message.


<!-- This is an auto-generated comment by OSS CodeRabbit -->